packages/webapp/pages/plus/index.tsx

```diff
@@ -17,12 +18,19 @@ const PlusMobile = dynamic(() =>
     /* webpackChunkName: "plusMobile" */ '@dailydotdev/shared/src/components/plus/PlusMobile'
   ).then((mod) => mod.PlusMobile),
 );
+
 const PlusDesktop = dynamic(() =>
   import(
     /* webpackChunkName: "plusDesktop" */ '@dailydotdev/shared/src/components/plus/PlusDesktop'
   ).then((mod) => mod.PlusDesktop),
 );
 
+const PlusIOS = dynamic(() =>
+  import(
+    /* webpackChunkName: "PlusIOS" */ '@dailydotdev/shared/src/components/plus/PlusIOS'
```

Comment:
My OCD got triggered
```suggestion
    /* webpackChunkName: "plusIOS" */ '@dailydotdev/shared/src/components/plus/PlusIOS'
```